Assets/Scripts/Player/PlayerScriptableObject.cs

```diff
@@ -7,6 +7,7 @@ public class PlayerScriptableObject : ScriptableObject
     {
         public int maxHealth;
         public float movementSpeed;
+        public float rotationSpeed;
```

Comment:
Why is this needed if the player is constantly looking towards the mouse position?

Reply:
correct